changelogs/fragments/44624-dnsmadeeasy-fix_locale.yaml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - dnsmadeeasy - force the date to be rendered with US locale as required by API
```

IMO US locale might scare people who know that they don't have it installed on their machine(s). In particular since you don't use the US locale at all, but the C locale :)
```suggestion
  - dnsmadeeasy - force the date to be rendered with C locale as required by API
```

---

Thank you for the hint. I have updated the changelog as well as the commit message.